### DISCUSSION TITLE
Welcome to falcon-metatrust Discussions!

### DISCUSSION THREAD

**Discussion by BradMoonUESTC** [MetaTrustLabs/falcon-metatrust#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vijendranagar** [comment]
While running falcon tool on a git repository I am getting assertion error can someone help me to understand and resolve this
Exact Error:```  File "/usr/local/lib/python3.10/dist-packages/falcon_analyzer-0.2.28-py3.10.egg/falcon/solc_parsing/solidity_types/type_parsing.py", line 200, in _find_from_type_name    return UserDefinedType(var_type)  File "/usr/local/lib/python3.10/dist-packages/falcon_analyzer-0.2.28-py3.10.egg/falcon/core/solidity_types/user_defined_type.py", line 19, in __init__    assert isinstance(t, (Contract, Enum, Structure))AssertionError
